api/src/main/java/org/eclipse/microprofile/config/package-info.java

```diff
@@ -82,5 +82,5 @@
  * @author <a href="emijiang@uk.ibm.com">Emily Jiang</a>
  * @author <a href="mailto:struberg@apache.org">Mark Struberg</a>
  */
-@org.osgi.annotation.versioning.Version("2.0.0")
+@org.osgi.annotation.versioning.Version("3.0.0")
```

Comment:
How about if we align these versions?

Reply:
@radcortez please see my comments [here](https://github.com/eclipse/microprofile-config/pull/699#discussion_r640165314)